src/apps/bridge/sensor_drivers/pmeWipeSensor.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "abstractSensor.h"
+#include "avgSampler.h"
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#define PME_WIPE_NUM_SAMPLE_MEMBERS 2
+
+typedef struct pme_wipe_aggregations_s {
+  double wipe_current_mean_ma;
+  double wipe_duration_s;
+  uint32_t reading_count;
+} pme_wipe_aggregations_t;
```

Comment:
Just FYI there are more fields incoming soon. You don't need to hold this PR for them, but PME is going to send more fields in the wipe message.